benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -2516,13 +2516,14 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2018",
+    "LiveFrom": "10/06/2022",
     "CaseTypeID": "Benefit",
-    "ID": "createTestCase",
-    "Name": "Create test case",
-    "Description": "Create a test case in validAppealCreated state",
```

Comment:
Do we not need this anymore ?

Reply:
I think caseevent is too large now, the last events have their dates parsed incorrectly. It's a test event added years ago so I assume it's safe to remove